### PR TITLE
Read and register all fonts in "font" directory

### PR DESCRIPTION
**Pull request by t-bullock**

### CODE REVIEW

File: pdfmaker/font_reader.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfbase.ttfonts import TTFont
+from fontTools import ttLib
```
Comment (jdaly101) [inline pdfmaker/font_reader.py:4]: Is fontTools for converting non-TTF fonts?

File: pdfmaker/font_reader.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from glob import glob
+
+
+def registerFonts():
+    rootDir = "fonts"
+
+    for fontLoc in glob(rootDir + "/*.ttf"):
```
Comment (jdaly101) [inline pdfmaker/font_reader.py]: If they have no ttf files in their font directory, it looks like this won't throw up an error and most likely blow up when it starts trying to draw glyphs.

File: pdfmaker/font_reader.py (hunk 1 above)
Comment (t-bullock) [inline pdfmaker/font_reader.py:4]: It's for grabbing the font name from a ttf. I can see if there's a way to do this without TTFont.

File: pdfmaker/font_reader.py (hunk 2 above)
Comment (t-bullock) [inline pdfmaker/font_reader.py]: Ok, this is now fixed

**Comment by kornaros** [comment]
Dear JDALY101 Could you please have a look at
http://www.mediafire.com/download/sarub5mk043ylm6/byzantinemusic-enfinal.pdf

and send me an email to me?(my email is in the documentation) 
I hope you will find byzantinemusic.sty XELATEX package ineteresting!
Charalampos

**Pull request merged**